docs/guidance/index.md

```diff
@@ -17,4 +17,4 @@ build
 publish
 ```
 
-Do you have questions or need assistance? Contact the [OCDS Helpesk](../support/index).
+Do you have questions or need assistance? Contact the [OCDS Helpdesk](../support/index) or you can fill out the [Training Request Form](http://bit.ly/OCP-Training).
```

Comment:
Thanks for correcting the typo. I think the link works better on this page: https://standard.open-contracting.org/latest/en/guidance/design/

Changing:

```md
Action: [Contact the OCDS Helpdesk](https://standard.open-contracting.org/latest/en/support/) to arrange an online introductory training for your technical team, in English or Spanish.
```

to:

```md
Action: [Contact the OCDS Helpdesk](https://standard.open-contracting.org/latest/en/support/) or [complete this form](http://bit.ly/OCP-Training) to arrange an online introductory training for your technical team, in English or Spanish.
```

Reply:
I agree, I'll apply the suggestion, thanks!